<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forget</class>
 <widget class="QDialog" name="Forget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>找回密码</string>
  </property>
  <widget class="QLineEdit" name="user_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_phone">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_email">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_question">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="enter">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>200</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
